docs/plugins/plugin-writer/basics.rst

```diff
@@ -64,3 +64,10 @@ plugin components, particularly for both models and viewsets.
 Subclassing Importer, Publisher
 -------------------------------
 
+Error Handling
+--------------
+
+Please see the :ref:`error-handling` section in the code guidelines.
+
+Non fatal exceptions should be recorded with :meth:`pulpcore.plugin.tasking.Task.append_non_fatal_error`
+These non-fatal exceptions will be returned as a non_fatal_errors attribute on the returned task.
```

Comment:
Same suggestion here for the attribute
"returned as a :attr:\`~pulpcore.app.models.Task.non_fatal_errors\` attribute"